be/src/runtime/memory/mem_tracker.h

```diff
@@ -154,13 +154,13 @@ class MemTracker {
     static void refresh_all_tracker_profile();
 
 public:
-    Snapshot make_snapshot() const;
+    virtual Snapshot make_snapshot() const;
```

Comment:
Why change it to `virtual`, where need to use `MemTrcker*` to call `MemTrckerLimiter*`